files/image_config/monit/container_checker

```diff
@@ -84,24 +66,40 @@ def get_current_running_containers():
     """
```

Comment:
Since the change will not use `docker ps` to get the running container list, can you reorganize the comments of this function please?